src/main/java/com/google/api/generator/engine/escaper/CommentEscaper.java

```diff
@@ -18,7 +18,7 @@
 import com.google.common.escape.Escapers;
 
 public class CommentEscaper {
-  private static class SpecialEscaper extends Escaper {
+  private static class SpecialCharEscaper extends Escaper {
```

Comment:
I would recommend splitting SpecialCharEscaper and HtmlEscaper into separate public classes, since there isn't anything inherently "comment"-y about them.

Reply:
Also, let's rename this particular escaper to something like "MetacharEscaper" or similar.

Reply:
Thanks for the advice, done.